Src/AutoFixture/InstanceGeneratorNode.cs

```diff
@@ -1,164 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Reflection;
-
-namespace Ploeh.AutoFixture
-{
-    /// <summary>
-    /// Default reusable implementation of <see cref="IInstanceGenerator" />
-    /// </summary>
-    public class InstanceGeneratorNode : IInstanceGenerator

```

Comment:
This file was removed in 53e22adc35a232be8e59841de9b83b8c02275136, but then accidentally (it seems) re-added in 2c81087a99d512f69ef239a03dde4200d4c99c65.

I'll remove it from *master* shortly.